### PR TITLE
Provide method for setting backupGlobals

### PR DESCRIPTION
**Pull request by mkeasling**
This address issue #3044 

Pass the backup_globals variable, if provided in the .yml file, to the test suite.

### CODE REVIEW

**Comment by DavertMik** [comment]
Ok, I got your point but I'd like to improve the implementation:

Please add `backup_globals` parameter to the configuration https://github.com/Codeception/Codeception/blob/2.1/src%2FCodeception%2FConfiguration.php#L88

So you won't need to check for `isset` as you always receive default values from the global config.

also could you add this parameter to reference? https://github.com/Codeception/Codeception/blob/2.1/docs%2Freference%2FConfiguration.md

**Comment by mkeasling** [comment]
Great feedback; thank you!

I've addressed your requests; please let me know if there's anything else I can do to improve this.

File: src/Codeception/SuiteManager.php [hunk 1]
```diff
@@ -142,6 +142,9 @@ protected function createSuite($name)
             $name = $this->settings['namespace'] . ".$name";
         }
         $suite->setName($name);
+        if (isset($this->settings['backup_globals'])) {
+            $suite->setBackupGlobals(boolval($this->settings['backup_globals']));
```
Comment (Naktibalda) [inline src/Codeception/SuiteManager.php]: boolval method was added in PHP 5.5, please use (bool) cast instead.

**Comment by DavertMik** [comment]
Thanks!

**Pull request merged**